templates/patient/insurance/insurance_edit.html.twig

```diff
@@ -0,0 +1,425 @@
+    }, {
+        name: "Country"|xl
+        ,required: true
+        ,data_type: country_data_type
+        ,field_id: "subscriber_country"
+        ,fld_length: 10
+        ,max_length: 63
+        ,smallform: 'true'
+        ,classNames: "js-capitalize-me mb-1 sel2"
+        ,list_id: country_list
+    }, {
+        name: "Subscriber Phone"|xl
+        ,required: false
+        ,data_type: dataTypeText
+        ,field_id: "subscriber_phone"
+        ,fld_length: 20
+        ,max_length: 63
+        ,smallform: 'true'
+        ,classNames: "mb-1"
+    }, {
+        name: "CoPay"|xl
+        ,required: false
+        ,data_type: dataTypeText
+        ,field_id: "copay"
+        ,fld_length: 6
+        ,max_length: 63
+        ,smallform: 'true'
+        ,classNames: "mb-1"
+    }, {
+        name: "Accept Assignment"|xl
+        ,required: true
+        ,data_type: dataTypeSelectList
+        ,field_id: "accept_assignment"
+        ,list_id: 'yesno'
+        ,smallform: 'true'
+        ,classNames: "mb-1 sel2"
+        ,empty_title: 'SKIP'
+    }]
+}
+%}
+    <div class="container-xl container-loading">
+        <div class="alert alert-info">
+            <h3>{{ "Loading..."|xlt }} <i class="wait fa fa-cog fa-spin ml-2"></i></h3>
+        </div>
+    </div>
+    <div class="container-xl container-loaded d-none">
+        <div class="row">
+            <div class="col-12">
+                <h2>{{ "Edit Current Insurance"|xlt }}</h2>
+            </div>
+            <div class="col-12">
+                <div class="btn-group text-align-right">
+                    <input type="button" class="btn btn-primary btn-add-policy" value="{{ 'Add New Policy'|xla }}" />
+                    <input type="button" class="btn btn-danger d-none btn-edit-return" value="{{ 'Return to Edit Screen'|xla }}" />
+                    <a href="./demographics.php" class="btn btn-secondary"><i class="fa fa-back"></i>{{ 'Back to Patient'|xlt }}</a>
+                </div>
+                <hr />
+            </div>
+        </div>
+        {# TODO: @adunsulag note that the form with id DEM here has a number of hard-coded styles which is where the insurance form gets its layout from. look at changing this. #}
+        <form id="DEM">
+            <div class="section-header">
+                <span class="text font-weight-bold">{{ "Insurance"|xlt }}</span>
+            </div>
+            {% include "patient/insurance/_insurance_edit_screen_new.html.twig" %}
+            {% include "patient/insurance/_insurance_edit_screen_edit.html.twig" %}
+        </form>
+    </div>
+    <script type="module">
+        // import { InsuranceEditController } from webroot + '/library/js/oeUI/insurance/InsuranceEditController.js';
+        // need to hit the server endpoint to grab the patient insurances for the given pid
+        // then I need to popualate the twig template with the data
+        let csrfToken = {{ csrfTokenRaw('api')|js_escape() }};
```

Comment:
not sure if this means anything, but i generally don't put a closed parenthesis `()` at the end of twigfilters when using them

Reply:
I can change it if you want.  The parenthesis are optional if the filter doesn't have any additional parameter settings.

Reply:
no need to change